apps/root/src/pages/position-detail/components/summary-container/index.tsx

```diff
@@ -32,50 +28,54 @@ const StyledFlexGridItem = styled(Grid)`
 interface PositionSummaryContainerProps {
   position: PositionWithHistory;
   pendingTransaction: string | null;
-  yieldOptions: YieldOptions;
-  totalGasSaved?: bigint;
+  yieldOptions?: YieldOptions;
+  isLoadingYieldOptions: boolean;
+  isLoading: boolean;
 }
 
 const PositionSummaryContainer = ({
   position,
   pendingTransaction,
   yieldOptions,
-  totalGasSaved,
+  isLoadingYieldOptions,
+  isLoading,
 }: PositionSummaryContainerProps) => {
   const currentBreakpoint = useCurrentBreakpoint();
 
   const isDownMd = currentBreakpoint === 'xs' || currentBreakpoint === 'sm';
   return (
-    <>
-      <Grid container spacing={4} alignItems="flex-start">
-        <StyledFlexGridItem item xs={12} md={5}>
-          <Sticky enabled={!isDownMd} top={95}>
-            <StyledPaper variant="outlined">
+    <Grid container spacing={4} alignItems="flex-start">
+      <StyledFlexGridItem item xs={12} md={5}>
+        <Sticky enabled={!isDownMd} top={95}>
+          <StyledPaper>
+            {isLoading ? (
+              <PositionDataSkeleton />
+            ) : (
               <Details
                 position={position}
                 pendingTransaction={pendingTransaction}
                 yieldOptions={yieldOptions}
-                totalGasSaved={totalGasSaved}
```

Comment:
I think we should still show this for eth mainnet

Reply:
This has been moved to position-data directly